src/main/java/net/neoforged/neoforge/common/conditions/IConditionBuilder.java

```diff
@@ -33,16 +35,36 @@ default ICondition or(ICondition... values) {
         return new OrCondition(List.of(values));
     }
 
+    default ICondition itemExists(ResourceLocation itemName) {
+        return new ItemExistsCondition(itemName);
+    }
+
     default ICondition itemExists(String namespace, String path) {
-        return new ItemExistsCondition(namespace, path);
+        return new ItemExistsCondition(ResourceLocation.fromNamespaceAndPath(namespace, path));
+    }
+
+    default ICondition itemExists(String itemName) {
```

Comment:
Remove this one? It's confusing I think, RL ftw!

Reply:
tbh ive been looking at this and asking myself why its strictly limited to items only?
conditions have a full registry access, could we expand this and tagEmpty to support any registry type?

say a mod wanted to do something if a enchantment exists, or block tag is empty. why limit them to strictly items?